README.md

```diff
@@ -1 +1,7 @@
-#401n1 Angular Template
+Brew Buddy Front End app
```

Comment:
Great job having a readme, but this should have a project description and perhapse
a link to the backend app